src/validators/block.validator.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ * Copyright 2019 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import {ExpectedError, ValidationContext, Validator} from 'clime';
+
+export class HeightValidator implements Validator<number> {
```

Comment:
The command looks good in overall. Could you please add a unit test for the ``HeightValidator``. Thanks!

Reply:
Ok, I will add unit tests in subsequent updates.

Reply:
@NemTechCN Ok, please ping me when this is done to merge the PR.

Reply:
@dgarcia360 Unit testing has been added